### DISCUSSION TITLE
What is the color theme used in the readme preview?

### DISCUSSION THREAD

**Discussion by Metaphysics0** [sxyazi/yazi#635] [1. Q&A]
Anyone know?
Looks amazing

I'm referring to this
<img width="824" alt="Screenshot 2024-02-06 at 15 32 19" src="https://github.com/sxyazi/yazi/assets/47161696/f1b5ee86-8749-4186-bfbe-1853cbd0eeda">

**Comment by sxyazi** [comment]
See [Catppuccin Macchiato](https://github.com/yazi-rs/themes?tab=readme-ov-file#catppuccin-macchiato)
